Cubical/Functions/Embedding.agda

```diff
@@ -163,6 +146,23 @@ isEmbedding≡hasPropFibers
            (λ _ → hasPropFibersIsProp _ _)
            (λ _ → isPropIsEmbedding _ _))
 
+-- We use the characterization as hasPropFibers to show that naive injectivity
+-- implies isEmbedding as long as B is an h-set.
+module _
+  {f : A → B}
+  (iSB : isSet B)
+  (inj : ∀{w x} → f w ≡ f x → w ≡ x)
+  where
+
+  injective→hasPropFibers : hasPropFibers f
+  injective→hasPropFibers y (x , fx≡y) (x' , fx'≡y) =
+    Σ≡Prop
+      (λ _ → iSB _ _)
+      (inj (fx≡y ∙ sym (fx'≡y)))
+
+  injEmbedding : isEmbedding f
+  injEmbedding = hasPropFibers→isEmbedding injective→hasPropFibers
+
```

Comment:
Mayb the stuff you deleted below, could still be a corollary here? I mean we could keep ```retractableIntoSet→isEmbedding``` but prove it using ```injEmbedding``` (and ```congRetract``` in a ```where```-block).

Reply:
I will look into it.

Reply:
Ok, I did it as you suggest, seems reasonable to me.